CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added default "none" for option shuffle in BatchGenerator, so that it works in executables where shuffle is not an option.
 - Added a few missing header files in shortlist.h and beam_search.h.
 - Improved handling for receiving SIGTERM during training. By default, SIGTERM triggers 'save (now) and exit'. Prior to this fix, batch pre-fetching did not check for this sigal, potentially delaying exit considerably. It now pays attention to that. Also, the default behaviour of save-and-exit can now be disabled on the command line with --sigterm exit-immediately.
+- Fix the runtime failures for FASTOPT on 32-bit builds (wasm just happens to be 32-bit) because it uses hashing with an inconsistent mix of uint64_t and size_t.
```

Comment:
We should include fix for binary model loading in the 32 bit. I realise this is my fault though

Reply:
@XapaJIaMnu Probably, I didn't understand. We already included [your fix](https://github.com/browsermt/marian-dev/pull/24/commits/e8ca9eacb253023235db0c83516d08f5959bd338) that solves this problem. Right? Do you mean just updating the CHANGELOG.md?

Reply:
Updated the changelog to reflect the things added/fixed/changed by this PR 👍